components/servo/lib.rs

```diff
@@ -540,16 +540,16 @@ where
                     .on_scroll_event(scroll_location, cursor, phase);
             },
 
-            EmbedderEvent::Zoom(magnification) => {
-                self.compositor.on_zoom_window_event(magnification);
+            EmbedderEvent::Zoom(zoom) => {
```

Comment:
Several places in this compositor.rs still use `magnification` as the name of the zoom factor. Maybe this can be left as magnification to be consistent?

Reply:
Sure. Though I'm not sure if long-term `magnification` is the right term.